egl/egl_util.c

```diff
@@ -30,6 +30,26 @@
 #include "libdrm/drm_fourcc.h"
 #endif
 
+static PFNEGLCREATEIMAGEKHRPROC createImageProc = NULL;
+static PFNEGLDESTROYIMAGEKHRPROC destroyImageProc = NULL;
+
+EGLImageKHR createImage(EGLDisplay dpy, EGLContext ctx, EGLenum target,
+                        EGLClientBuffer buffer, const EGLint *attrib_list)
+{
+    if (!createImageProc) {
```

Comment:
Not ready get your point. But I think at least we can make  imageTargetTexture2DProc init in start time.Because it will called in every frame.